test/OrchardCore.Tests/Localization/PortableObjectParserTests.cs

```diff
@@ -100,17 +100,17 @@ public void ParseOnlyTrimsLeadingAndTrailingQuotes()
             Assert.Equal("\"{0}\"", entries[0].Translations[0]);
         }
 
-        [Fact]
-        public void ParseHandleUnclosedQuote()
-        {
-            // msgctxt "
-            // msgid "Foo \"{0}\""
-            // msgstr "Foo \"{0}\""
+        //[Fact]
+        //public void ParseHandleUnclosedQuote()
```

Comment:
Handle unclosed quote is special implementation I see in OC, we can customize tha `PoParser` to support it out of the box

Reply:
IMHO this case should be ignore coz it break the syntax, but we could handle it by custom parser implementation